backend/api/blogpost/models.py

```diff
@@ -1,13 +1,15 @@
 from django.db import models
+from datetime import date
 
 # Create your models here.
 class Blogpost(models.Model):
 	# title = models.CharField(max_length=255, null=False)
 	# body = models.CharField(max_length=1000, null=False)
-	media_url = models.CharField(max_length=200, null=True)
+	media_url = models.CharField(max_length=200, blank=True) #we may not use this and instead use embeddings in the
+	# text of the blogpost
 	# TODO: when users table is created, make this a foreign key
 	author_id = models.IntegerField(null=False) # eventually this will be linked to the users table
-	posted_on = models.DateField(null=False)
+	posted_on = models.DateField(null=True)
```

Comment:
I'm having it auto-populate if the field is empty. If I don't set null=true, it rejects the attempt to create an object before it even gets to the save() function that populates the field because of validation reasons.